### PR TITLE
:lipstick:

### PR DESCRIPTION
**Pull request by SamVerschueren**
- XO
- AVA
- Node 4 + 5
- Added documentation for the input params (need some review)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -33,10 +33,22 @@ lpad(str, '    ');
...
-### lpad(string, pad)
+### lpad(input, pad)
 
 Pads each line in a string with the supplied string.
 
+#### input
+
+Type: `string`
+
+Multiline string that will be padded.
```
Comment (sindresorhus) [inline readme.md]: It support single-line strings too, so just drop the `multiline` word.

File: test.js [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
-var lpad = require('./');
+import test from 'ava';
+import m from './';
 
-it('should pad string', function () {
-	assert.equal(lpad('foo\nbar', '    '), '    foo\n    bar');
-	assert.equal(lpad('foo\r\nbar', '    '), '    foo\n    bar');
-	assert.equal(lpad('foo\nbar\n', '    '), '    foo\n    bar\n');
-	assert.equal(lpad('foo\n\nbar', '    '), '    foo\n\n    bar');
+test('pad string', t => {
```
Comment (sindresorhus) [inline test.js]: No need for a title since there's only one test.

File: readme.md [hunk 3]
```diff
@@ -33,10 +33,22 @@ lpad(str, '    ');
...
+
+#### pad
+
+Type: `string`
+
+String that will be prepended to each line.
+
 
 ## License
```
Comment (sindresorhus) [inline readme.md:54]: `http` => `https` ⬇️

**Comment by SamVerschueren** [comment]
Fixed

**Comment by sindresorhus** [comment]
Looks good :) 🦄

**Comment by SamVerschueren** [comment]
Nice!

**Pull request merged**